tests/test_xml.py

```diff
@@ -6,22 +6,14 @@
 import stat
 import tempfile
 
-from xml.etree import ElementTree
-try:
-    from xml.etree.ElementTree import ParseError
-except ImportError:
-    from xml.parsers.expat import ExpatError as ParseError
-
```

Comment:
this is needed for XML generation. Removal will have an affect to module generation.